app/src/js/components/Table/Table.js

```diff
@@ -1,7 +1,6 @@
-import React, { useState, useEffect, lazy, Suspense, useCallback, useRef } from 'react';
+import React, { useState, useEffect, useMemo, lazy, Suspense, useCallback, useRef } from 'react';
```

Comment:
Similar question on this file. There seem to be a lot of changes here related to memoization on tables. Was this change related to the Router update (or a knock-on effect of that update)? What drove these changes?

Reply:
Because we have links to routes in our tables I had to be able to use the hooks for React Router then I had to memorize the hooks so the queries wouldn't endlessly loop.